Automattic/simplenote-android
# Update Edit Tags

### Fix
Update the list item layout on the ***Edit tags*** screen.  These changes include:
- Add minimum height of row to 48dp
- Add trash button hint message on long-press
- Remove horizontal dividers between list items
- Remove vertical dividers between count text and trash button

See the screenshots below for illustration.

![update_edit_tags](https://user-images.githubusercontent.com/3827611/65711561-5cc70e00-e052-11e9-86e0-44950b92359c.png)

### Test
1. Open navigation drawer.
2. Tap ***Edit*** button in ***Tags*** group.
3. Notice updated list item layout.
4. Long-press trash button.
5. Notice ***Delete tag*** message.

### Review
Only one developer and one designer are required to review these changes, but anyone can perform the review.

### Release
These changes do not require release notes.
